### DISCUSSION TITLE
2005/11/07/net-framework-%E7%9A%84%E5%AE%B9%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#54] [Announcements]
# 2005/11/07/net-framework-%E7%9A%84%E5%AE%B9%E5%99%A8

自从我发现了STL，再编写C++程序，就离不开它了。STL提供的容器和算法极大的方便了C++编程。

https://blog.qizhen.xyz/2005/11/07/net-framework-%E7%9A%84%E5%AE%B9%E5%99%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年09月27日 20:15 编辑
改好了，多谢提醒。

回复
Hao
2008年09月27日 16:01 编辑
Queue与 STL中的 Queue 相类似。是元素先进后出的集合。Stack与 STL中的 Stack 相类似。是元素先进先出的集合。
 
Queus是先进先出，Stack是先进后出。

回复
璐
2005年11月13日 14:10 编辑
路过：）

回复
璐
2005年11月13日 14:09 编辑
路过：）

回复
Unknown
2005年11月9日 11:51 编辑
祝贺您的空间已上传M友中文论坛首页的精彩空间荟萃，请您查看信息和分类是否正确。www.cnmsnbbs.com有需要调整者请在空间登记处栏目回复。同时敬请在您的空间链接M友中文论坛，欢迎注册论坛并参与论坛各栏目创作。

回复
liangliang
2005年11月7日 20:09 编辑
好深奥……看不懂~干嘛在我的计数器那里留言阿？>_<

回复
